### DISCUSSION TITLE
is svelte ux working?

### DISCUSSION THREAD

**Discussion by colozzacristian** [symfony/symfony#58276] [Q&A]
Today i tried to use the ux-svelte package, but npm and yarn can't seem to find the package, and composer does not automatically add the dependency(\@symfony/ux-svelte) to my json.lock file.
I can't find anyone else encountering this problem, and everything i tried didn't work.

**Comment by alexisLefebvre** [comment]
Please share the error messages.

**Reply by colozzacristian** [reply]
Unfortunately i don't have a screen of it right now, but how i already said, nor npm nor yarn could install the required package, the resulting errors where caused by this first one. 
Are you able to install the package on your machine?

**Reply by alexisLefebvre** [reply]
I don't use Svelte but I'll try to help.

Did you follow this documentation? https://symfony.com/bundles/ux-svelte/current/index.html What step doesn't work?

**Reply by colozzacristian** [reply]
Yes i did follow it. but it does not work as intended, as when i tried to "npm run dev" it could not import the package specified in my first message.
